src/app/assets/containers/asset-detail/asset-detail.component.html

```diff
@@ -52,10 +52,20 @@ <h2>{{theAsset.name}}</h2>
     <div class="asset-details-card mat-elevation-z8" *ngFor="let theDefendant of (defendants$ | async)">
       <app-view-defendant
         [defendant]="theDefendant"
+        *ngIf="getPropertyState(theDefendant.id) === 'view'"
```

Comment:
Can we create a method in the component in order to not hardcode strings, especially in the template, please?